src/scripts.js

```diff
@@ -95,27 +95,19 @@ function randomizeUser(data) {
 // ~~~~~~~~~~~~~~ Main View Functions ~~~~~~~~~~~~~~~~~~~~
 function displayHomePage() {
     allRecipes.innerHTML = ''
-    hide(removeRecipeButton)
-    show(allRecipes)
-    hide(singleRecipe)
-    hide(favoritesView)
-    hide(favoriteRecipeButton)
-    show(favoriteButton)
-    show(filterSidebar)
-    hide(ingredientSidebar)
+    const hideElementsList = [removeRecipeButton, singleRecipe, favoritesView, favoriteRecipeButton, ingredientSidebar]
+    const showElementsList = [allRecipes, favoriteButton, filterSidebar]
+    hide(hideElementsList)
+    show(showElementsList)
     displayAllRecipes()
     homeView = true
 }
 
 function displayFavoritesPage() {
-    hide(removeRecipeButton)
-    hide(allRecipes)
-    hide(singleRecipe)
-    show(favoritesView)
-    hide(favoriteRecipeButton)
-    hide(favoriteButton)
-    show(filterSidebar)
-    hide(ingredientSidebar)
+    const hideElementsList = [removeRecipeButton, allRecipes, singleRecipe, favoriteRecipeButton, favoriteButton, ingredientSidebar]
+    const showElementsList = [favoritesView, filterSidebar]
+    hide(hideElementsList)
+    show(showElementsList)
```

Comment:
I think we can dry this up and don't need the variables and just have `hide([removeRecipeButton, allRecipes, singleRecipe, favoriteRecipeButton, favoriteButton, ingredientSidebar])`
and `show([favoritesView, filterSidebar])`

Reply:
Same goes for all the others, it seems like this variable is function scoped and we aren't using it elsewhere